TrackingTools/GsfTools/plugins/BuildFile.xml

```diff
@@ -1,3 +1,4 @@
+<flags   CXXFLAGS="-Ofast"/>
```

Comment:
maybe remove this, hoping to remove changes in RECO products and simplify integration?
... I didn't look if anything else could change the RECO products in this PR